ddns-route53

```diff
@@ -51,7 +51,7 @@ function fetch-current-ip() {
   aws route53 list-resource-record-sets \
     --hosted-zone-id "$zone_id" \
     --output text \
-    --query "ResourceRecordSets[?Name == '$record_set.' && Type == '$record_type'].ResourceRecords[0].Value | [0] || ''"
+    --query "ResourceRecordSets[?Name == '$record_set.' && Type == '$record_type'].ResourceRecords[0].Value"
```

Comment:
Hmm, can't remember why I added `| [0] | ''` as it seems to work without it :thinking:.

Reply:
yes, it doesn't seem to work with it..